koswat/cost_report/infrastructure/infrastructure_location_profile_cost_report.py

```diff
@@ -31,7 +31,7 @@ def location(self) -> PointSurroundings | None:
     @property
     def total_cost(self) -> float:
         if not self.infrastructure_location_costs:
-            return 0
+            return 0.0
```

Comment:
this should not be needed?

Reply:
It will return an `int` which will show up on the report. Returning `0.0` will force it to be a float.